### PR TITLE
Project Updates (second_path & relocate flatcar)

### PR DESCRIPTION
**Pull request by mrbobbytables**
Dedupes projects that are located in multiple categories and switches them to using `second_path`.

Moves flatcar to Platform category so that  it is accounted for on the website correctly.

### CODE REVIEW

**Comment by mrbobbytables** [comment]
ah woops, thought I go tall of the ones that need deduping - there are a few others. Let me take care of those before this PR gets merged

**Comment by mrbobbytables** [comment]
theres a lot more to dedupe than I thought, so this will have to be a first pass for now - important one fixed is flatcar 

cc @caniszczyk @cjyabraham @tegioz

File: landscape.yml [hunk 1]
```diff
@@ -8034,6 +8072,20 @@ landscape:
...
             crunchbase: https://www.crunchbase.com/organization/flant
+          - item:
+            name: Flatcar Container Linux
+            description: >-
+              A community Linux distribution designed for container workloads, with high security and low maintenance
+            homepage_url: https://www.flatcar.org/
+            repo_url: https://github.com/flatcar/Flatcar
+            logo: flatcar.svg
+            project: incubating
+            twitter: https://twitter.com/flatcar
```
Comment (cjyabraham) [inline landscape.yml:8083]: Don't you need a `second_path` entry here to point to the category you're removing it from below?

Comment (mrbobbytables) [reply landscape.yml:8083]: weird, I thought I had - but guess I missed it - will add it now quick

Comment (mrbobbytables) [reply landscape.yml:8083]: Updated :+1:

File: landscape.yml [hunk 2]
```diff
@@ -8034,6 +8072,22 @@ landscape:
...
+            second_path:
+              - "Wasm / Edge/Bare Metal"
```
Comment (tegioz) [inline landscape.yml:8085]: Can you update `Bare Metal` to `Bare metal` (metal `m` lowercase)? Categories and subcategories names are case sensitive.

**Review by tegioz — CHANGES_REQUESTED** [review]
Thanks @mrbobbytables!

This is great and it will solve some issues 💯

**Comment by tegioz** [comment]
We could also add an entry to the [PR template](https://github.com/cncf/landscape/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to encourage folks to review if any of the items they'd like to add already exists, and use the second path approach when applicable instead of duplicating.

This should only be used when duplicating items for a different group (i.e. Serverless, Wasm, etc), not to add the same item to multiple categories in the same group (something we'll need to keep an eye on while reviewing PRs).

File: landscape.yml (hunk 2 above)
Comment (mrbobbytables) [reply landscape.yml]: done :+1:

**Comment by mrbobbytables** [comment]
Agreed re:PR template, should update that with some other general things like adding other repos etc.

**Pull request merged**